core/merginapi.cpp

```diff
@@ -2592,6 +2650,24 @@ bool MerginApi::isInIgnore( const QFileInfo &info )
   return sIgnoreExtensions.contains( info.suffix() ) || sIgnoreFiles.contains( info.fileName() );
 }
 
+bool MerginApi::excludeFromSync( const QString &filePath, const MerginConfig &config )
+{
+  if ( config.selectiveSyncEnabled )
+  {
+    QFileInfo info( filePath );
+    bool isImage = sIgnoreImageExtensions.contains( info.suffix() );
+    if ( config.selectiveSyncDir.isEmpty() )
+    {
+      return isImage;
```

Comment:
In this case `isImage` is always false, right? Can we rather return `false` explicitly?

Reply:
Maybe that condition can go to the top

Reply:
If `selectiveSyncDir` is empty thus is not set, then the whole project dir is taken by default. In that case is it enough to check only if extension is from the exclude extension list.
If its defined, beside extension condition there is also an extra check for a location

Reply:
Ok cool, I see now. Can you please add comment there stating how it behaves?